src/main/java/org/broadinstitute/consent/http/resources/DataRequestElectionResource.java

```diff
@@ -82,7 +84,8 @@ public Response createDataRequestElection(@Context UriInfo info, Election rec,
             }
             return createExceptionResponse(e);
         }
-        return Response.created(uri).entity(accessElection).build();
+        List<Election> elections = Arrays.asList(accessElection, rpElection);
+        return Response.created(uri).entity(elections).build();
```

Comment:
If we make the logical jump to returning all elections created due to the creation of an access election, then there is also the possibility of a data owner election. Those are created in one of the service methods if the dataset has a data owner and is marked as requiring data owner approval.

Reply:
I disagree with this approach conceptually. An endpoint should be focused on the response based on what it is doing. If we need an API to get ALL elections from a created dar election, that should be a separate endpoint. Please revert this logic, we need to have a larger design discussion before making changes like this.

Reply:
I think this response is more clearly based on what it is doing than returning only the access election which gives the false impression that was the only election created. But this method produces both, access and rp. Also, this endpoint is only used in one place where it would be beneficial to see the results of the two created elections (and not the data owner election, which may or may not exist). See my 1555 draft PR on the UI. If we want to talk about the design, I would be happy to talk today, also I would like some help trying to get my 1555 PR on the UI to connect and run with the new API, if you have time.


> An endpoint should be focused on the response based on what it is doing.